tests/integration/clipboard/test_clipboard.py

```diff
@@ -0,0 +1,48 @@
+import dash
+import dash_core_components as dcc
+import dash_html_components as html
+
+from selenium.webdriver.common.action_chains import ActionChains
+from selenium.webdriver.common.keys import Keys
+
+
+def test_clp001_clipboard_text(dash_dcc):
+    copy_text = "Hello, Dash!"
+    app = dash.Dash(__name__, prevent_initial_callbacks=True)
+    app.layout = html.Div(
+        [
+            html.Div(copy_text, id="copy"),
+            dcc.Clipboard(id="copy_icon", target_id="copy"),
+            dcc.Textarea(id="paste"),
+        ]
+    )
+    dash_dcc.start_server(app)
+
+    dash_dcc.find_element("#copy_icon").click()
+    dash_dcc.find_element("#paste").click()
+    ActionChains(dash_dcc.driver).key_down(Keys.CONTROL).send_keys("v").key_up(
+        Keys.CONTROL
+    ).perform()
+
+    elem = dash_dcc.find_element("#paste")
+    val = elem.get_attribute("value")
+    assert val == copy_text
```

Comment:
Now both tests are failing on this assertion... either one of these operations is disallowed in the CI environment or it's slower here and the pasted value hasn't shown up yet. I don't think it's slower, because both tests on all Python versions are consistently showing this same error and anyway we do essentially the same stuff in dash-table tests. Same reasoning says paste is not the problem, so the only thing left AFAICT is the clipboard API being blocked.

Here's a stackoverflow post that looks like the same issue: https://stackoverflow.com/questions/53669639/enable-clipboard-in-automated-tests-with-protractor-and-webdriver

Unfortunately the fix looks like it would need to be [deep inside dash.testing](https://github.com/plotly/dash/blob/597dc22c86de97530fb9b0a9bd9f7876bd0d7601/dash/testing/browser.py#L432-L434), I don't see a way to do it locally here.

So if you see these tests passing locally, I'm comfortable skipping them on CI for now and opening an issue in the main dash repo to follow up. You can do this with something like:

```py
@pytest.mark.skipif(
    'CIRCLECI' in os.environ,
    reason="clipboard API is blocked on CI, see <dash issue url>"
)
def test_clp001...
```